### PR TITLE
WIP: Add WebView to Insert Command from Command Explorer

### PR DESCRIPTION
**Pull request by corbob**
## PR Summary

Add a WebView for Inserting command selected from Command Explorer.

Currently mimics `Show-Command` in that it handles switches and text (switches don't get $true appended, and text is encapsulated in quotation marks if a space is involved).

WIP: `Show-Command` handles `ValidateSet` with a drop down. Would like to add this functionality before merging.

## PR Checklist

Note: Tick the boxes below that apply to this pull request by putting an `x` between the square brackets.
Please mark anything not applicable to this PR `NA`.

- [x] PR has a meaningful title
- [x] Summarized changes
- [ ] PR has tests
- [ ] This PR is ready to merge and is not work in progress
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready

### CODE REVIEW

**Comment by TylerLeonhardt** [comment]
Will review after I get back from vacation. P.S. I would really recommend attempting to add tests. It may be possible to mock the LSP requests to PSES.

**Comment by TylerLeonhardt** [comment]
@corbob can you add some pictures or a gif of what this looks like since it's UI related?

File: media/insert.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        <h1 id="CommandName"></h1>
+        <h2>Parameter Sets</h2>
+        <div id="ParameterSets"></div>
+        <input type="submit" />
+    </form>
+    <script src="%SCRIPTSRC%"></script>
+
+</body>
+
+</html>
```
Comment (TylerLeonhardt) [inline media/insert.html]: nit: always have a newline at EOF
```suggestion
</html>

```

File: media/style.css [hunk 2]
```diff
@@ -0,0 +1 @@
+/* TODO: Add some styles ¯\_(ツ)_/¯ */
```
Comment (TylerLeonhardt) [inline media/style.css]: nit newline ;)
```suggestion
/* TODO: Add some styles ¯\_(ツ)_/¯ */

```

**Comment by TylerLeonhardt** [comment]
Generally I think it might be worth thinking about how you could write tests for this.

My initial thought is, the InsertCommand method could be tested.
If you refactor the BuildCommand function a bit to return the built command, you could test that.

If you look at some of the CustomView tests, you can get an idea for how to reach in and call private methods.

**Comment by corbob** [comment]
Given that we're approaching 2 years since I started this PR and I don't foresee myself having the time to work on it anytime soon, I'll close these PRs.

I'll keep my branches around in case anyone wants to pick up the work and use them as a starting place (be forewarned: there are multiple merge conflicts you'll have to resolve to bring it up to date).

**Comment by andschwa** [comment]
That's fair, and being very new I'd have to start from scratch understanding the code. I'm sorry your contribution didn't make it in, but thank you anyway!

**Pull request closed without merging**